packages/admin/admin-rte/src/core/translation/stateToHtml.ts

```diff
@@ -0,0 +1,39 @@
+import { EditorState } from "draft-js";
+import { stateToHTML } from "draft-js-export-html";
+
+import { IRteOptions } from "../Rte";
+
+export function stateToHtml({ editorState, options }: { editorState: EditorState; options: IRteOptions }) {
+    const contentState = editorState.getCurrentContent();
+
+    const customInlineStyleKeys = options?.customInlineStyles ? Object.keys(options.customInlineStyles) : [];
+
+    const inlineStyles: any = {
+        SUB: { element: "sub" },
+        SUP: { element: "sup" },
+    };
+
+    customInlineStyleKeys.forEach((item) => {
+        inlineStyles[item] = { element: "span", attributes: { style: [item] } };
+    });
+
+    const linkDataList: { id: string; data: any }[] = [];
+
+    const html = stateToHTML(contentState, {
+        inlineStyles,
+        entityStyleFn: (entity) => {
+            const entityType = entity.getType();
+            const data = entity.getData();
+
+            if (entityType === "LINK") {
+                const id = `${linkDataList.length}`;
+                linkDataList.push({ id, data });
+                return { element: "a", attributes: { id } };
+            }
+
+            return undefined;
```

Comment:
We should throw an error for other entity types, since we don't support them in our editor.

Reply:
d1be9215e33466ef0387009f3a3cd6cb0678fbe4